.github/workflows/production.yml

```
@@ -3,32 +3,37 @@ name: Production
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Generate GitHub App Token
+        id: app-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.APP_PRIVATE_KEY }}
+
       - name: Set up Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)